### DISCUSSION TITLE
where to download a working version of the Flux safe tensors

### DISCUSSION THREAD

**Discussion by PeterJared** [lllyasviel/stable-diffusion-webui-forge#1071] [General]
The reason why I ask is I got a couple of safe tensors version that give me this error after a clean install from this repository:
TypeError: 'NoneType' object is not iterable

But then I installed it in Pinokio and it downloaded automatically a dev safe tensor of 11gb (the ones I tested were the double of size) and this works fine in Pinokio and if I copy it in the original installation it also works fine.
I'm missing the Schnell Flux version as the one I got get that error.

**Comment by Bullseye110** [comment]
That isn't an error from flux, but from Forge. Guess it's solved now. Or at least that's what the dev said.

**Comment by PeterJared** [comment]
afer getting last update, it simply gives a differn error message: 
AssertionError: You do not have CLIP state dict!

The 11gb Flux safe tensors, dev and Schnell,   from Pinokio work

**Reply by HMRMike** [reply]
You need to manually load clip, vae and t5 if the checkpoint doesn't have some of them baked in. Sometimes all three are missing. This way you can also load other versions instead of baked ones.
See here: https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/1050
